src/pages/Home/index.js

```diff
@@ -62,7 +67,17 @@ const Home = () => {
       where: [position.coords.latitude, position.coords.longitude],
     });
   };
+  const getMatches = () => {
```

Comment:
Since you're not transforming the data inside of map, you can just simply use the array
```suggestion
  const getMatches = () => {
    const matches = matchConversations?.matches || []
    setMatchList(matches);
  };
```